src/main/Manager.ts

```diff
@@ -127,7 +128,8 @@ export default class Manager {
       valid: false,
       current: this.obsBaseCfg,
       get: (cfg: ConfigService) => getObsBaseConfig(cfg),
-      validate: async (config: ObsBaseConfig) => Manager.validateBaseCfg(config),
+      validate: async (config: ObsBaseConfig) =>
```

Comment:
I think your prettier is somehow ignoring the `/* eslint-disable prettier/prettier */` above here?

Reply:
Let me check on that.